static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,190 @@
+
+- Tracked files can be safely stored in a **centralized** remote location, with
+  the ability to create any amount of distributed copies on other remote
+  storage.
+- Several projects can **share** the same files, trusting that everyone is using
+  the same data versions.
+- Projects that import data from the registry don't need to push these large
+  files to their own [remotes](/doc/command-reference/remote), **saving space**
+  on storage – they may not even need a remote at all, using only their local
+  <abbr>cache</abbr>.
+- It may be easier to manage **access control** for remote storage configured in
+  a single data registry project. A possible setup would use a read-only remote,
+  so other projects can't affect each other accidentally.
+
+A possible risk of shared data registries is that, if the source project or its
+remote storage are lost for any reason, several other projects depending on it
+may stop being reproducible. So this strategy is best when the registry is owned
+and controlled internally by the same team as the projects that employ it.
+
+## Implementing proper data versioning
+
+In the [Versioning Tutorial](/doc/tutorials/versioning) we use a ZIP file
+containing images of cats and dogs, and then we update the dataset with more
+images from a second ZIP file. These compressed archives are downloaded from our
+own
+[iterative/dataset-registry](https://github.com/iterative/dataset-registry)), a
+<abbr>DVC project</abbr> hosted on GitHub. They can be found as
+<abbr>outputs</abbr> of 2 separate
+[DVC-files](/doc/user-guide/dvc-files-and-directories) in the `tutorial/ver`
+directory.
+
+There are a few possible problems with the way this dataset is stored (in 2
+parts, as compressed archives). One is that dataset partitioning is an
+unnecessary complication; It can cause data duplication and other hurdles.
+Another issue is that extra steps are needed to bundle and extract file
+archives. The data compression also raises questions. But most importantly, this
+single dataset is tracked by 2 different DVC-files, instead of 2 versions of the
+same one, as we'll explain next.
+
+Let's download and extract the first archive we discussed above (in an empty
+directory) to visualize the structure of the dataset:
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry \
+          tutorial/ver/data.zip
+$ unzip -q data.zip
+$ tree --filelimit 3
+.
+├── data
+│   ├── train
+│   │   ├── cats [500 entries ...]
+│   │   └── dogs [500 entries ...]
+│   └── validation
+│       ├── cats [400 entries ...]
+│       └── dogs [400 entries ...]
+└── data.zip
+
+7 directories, 1 file
+$ rm -f data.zip
+```
+
+Instead of creating an archive containing the `data/` directory, we can simply
+put it under DVC control as-is! This is done with the `dvc add` command, which
+accepts entire directories. We can then save this first version using Git:
+
+```dvc
+$ git init  # Initialize Git repository
+$ dvc init  # Initialize DVC project
```

Comment:
project -> data registry?

Reply:
Hmm. The data registry is a DVC project. Will probably end up removing these commands anyway...